File: 2023/java/src/test/java/org/frieder/aoc/day1/TestDay1B.java
package org.frieder.aoc.day1;

import org.frieder.aoc.day1.b.Solution1B;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay1B
{

    @Test
    public void testExample() throws IOException {
        assertEquals(281, Solution1B.getResult("../resources/1/test_b.txt"));
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(56017, Solution1B.getResult("../resources/1/input.txt"));
    }
}

File: 2023/php/test/TestDay1B.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day1\b\Solution1B;
use PHPUnit\Framework\TestCase;

final class TestDay1B extends TestCase
{

    public function testExample(): void
    {
        $this->assertSame(281, Solution1B::getResult("../resources/1/test_b.txt"));
    }

    public function testResult(): void
    {
        $this->assertSame(56017, Solution1B::getResult("../resources/1/input.txt"));
    }

}